stdlib/public/core/Availability.swift

```diff
@@ -17,11 +17,8 @@ import SwiftShims
 ///
 /// This is a magic entry point known to the compiler. It is called in
 /// generated code for API availability checking.
-/// Note: It is important not to make this function inlinable. There is a pass
-/// that relies on being able to tell whether this function is called. It does
-/// this using the semantics attribute.
```

Comment:
Should we replace this comment with one that says the call to `_swift_stdlib_operatingSystemVersion` must be present, just in case we want to change that one day?